ansible/playbooks/postflight.yml

```
@@ -5,3 +5,17 @@
   become_method: sudo
   roles:
     - postflight
+
+- name: Change the state info of the previous epicli execution
+  hosts: all
+  gather_facts: true
```

```suggestion
  gather_facts: false
```
It looks like facts are not used here but gathering them takes time.

---

This would make sense if we update not only the `state` but the `date` as well.